setup_eunit.template

```diff
@@ -1,6 +1,6 @@
 {variables, [
     {package_author_name, "The Apache Software Foundation"},
-    {cluster_port, 5984},
+    {cluster_port, 0},
```

Comment:
As mentioned before, the backend port can't change unless some config_tests are fixed, which may be currently beyond the scope I'm trying to achieve. Also, since the dev cluster remaps the individual backend ports to e.g.
{{"httpd","port"},"15986"}
and they are not load balanced, it's ok to just change the cluster port, (which otherwise would collide with haproxy's listen port).